### ISSUE TITLE
Training data and scripts?

### ISSUE THREAD

**Issue by slowkow** [ncborcherding/Trex#7]
How did you create `Trex.Data`?

I can see in the comments that `Trex.Data` has "Amino acid properties and ranges for the autoencoder models along tensor", but what is the format of the data in this file and what is the source of the original data? Do you share scripts for creating this file? Do you have any references for the original publications that might be associated with the original data?

How did you create the .h5 files like `TRA_AF_Encoder.h5`? What data was used to create these h5 files? Does the Trex repository include scripts for creating these h5 files?

**Comment by ncborcherding** [comment]
Hey Kamil, 

Huge fan of your work. We are under review for publication and will be making all the training code public asap. Data was assembled across a publicly available 5' VDJ TCR sequencing experiments on GEO - I will assemble a list and associated citations. For the amino acid properties, a 10 residue amino acid sequence was transformed into a 10x5 matrix (for the Acthley factors) then linearized into a single vector (totally 350, with 0 padding up to 70 aa residues).

The .h5 files are the encoding portion of the compiled keras model and is based on the above scraped GEO data. The Trex repo will not include the scripts for creating it, there will be a secondary repo released as soon as everything is near publication (sorry for the delay). 

Please let me know of if you have other questions or suggestions (I like the data links idea).

Thanks,
Nick
